demo/pom.xml

```diff
@@ -75,4 +74,63 @@
             </plugin>
         </plugins>
     </build>
+    
+    <profiles>
+		<profile>
+			<id>default-jdk-demo</id>
+			<activation>
+				<jdk>(,1.8]</jdk>
+			</activation>
+			<modules>
+        		<module>appserver</module>
+			</modules>
+            <properties>
```

Comment:
Is there a mixture of tabs/spaces here? Can we normalise to just spaces?

Reply:
Note this applies to multiple poms :-)

Reply:
I think I updated them all now